GraphQL-TSC.md

```diff
@@ -162,24 +168,44 @@ When something cannot be decided by consensus or a formal vote is required, the
 
-#### Ensuring efficient votes
+A quorum of more than half (or two-thirds for a supermajority vote) of the TSC *attending* members must cast a ballot in order for a vote to be valid.
 
-A TSC member must attend two of the previous five meetings to be considered a "TSC voting member" and be eligible to vote. Should a TSC member miss three or more of the prior five meetings, they will no longer vote or be counted toward the total number of voting members. This is to ensure broad participation as well as to avoid voting deadlocks.
+### Voting process
 
-A member may regain their voting status by attending two meetings and may participate in votes in the second meeting.
+Because we work in a distributed environment, the voting process must account for a range of time zones and schedules. Once the threshold of a voting quorum has been met and a vote is valid, one of these two critera must be satisfied to conclude a vote:
 
-#### Resignation or removal of TSC members
+* A notice is sent via email that the vote will conclude in three business days, reminding those who haven't voted that they should do so. The vote will conclude at the end of this time.
+* The election results would not change with the addition of another TSC member's vote.
 
-A TSC member may voluntarily resign at any time before the end of their term by notifying the TSC chair. A TSC member may also be involuntarily removed by a supermajority (2/3) vote by eligible TSC voting members less any with a conflict of interest.
+Once a valid vote is concluded, the vote outcome is determined as follows:
 
-After a TSC seat vacancy the remaining TSC will collect self-nominations and vote on a replacement representative to complete the term. The nomination period should be open for at least two weeks, after which a vote shall be held.
+* For a simple majority (or supermajority), the votes in favor must exceed 50% (or 66%) of the total of, as of the time the vote is concluded:
+  * the number of TSC *attending* members.
+  * the number of members who are not considered a TSC *attending* member but did cast a vote.
+* For the Condorcet method, all votes received at the time the vote is concluded are considered.
+
+<details>
+<summary>Example</summary>
+
+Consider this scenario for a simple majority vote:
+
+* 6 of 11 TSC members are considered *attending* members by having attended at least two of the last five meetings.
+* Being a simple majority, more than half of this 6 (so 4 or more) must cast ballots to consider the vote valid. 4 ballots are cast.
+* An email is sent out reminding more members to cast a vote within three business days.
+* 1 additional TSC *attending* member casts a vote.
+* 2 members who are not considered an *attending* member also cast a vote.
+* The vote is concluded.
+* A total of 7 ballots have been cast
+* A majority is determined relative to the number of *attending* members (6) plus the number of non-attending but voting members (2), totaling 8.
```

Comment:
I don't know why this was confusing to me on first read, but given the numbers provided I was able to reverse engineer the algorithm to mean what it says (the base 100% number used to evaluate whether majority is reached is all attending members, voting or not, plus any voting but not attending member).

It's in the example so it's not required to be editorially perfect, and also after reading 2-3 times it's very clear and I can't un-realize again to figure out what wording would be more clear without muddying the example.

Reply:
I had the same process.

Reply:
Yeah :/ I think I'm missing a defining term for this